kolibri/core/assets/src/utils/minimumBrowserRequirements.js

```diff
@@ -1,16 +1,30 @@
+import isUndefined from 'lodash/isUndefined';
+import browsers from 'browserslist-config-kolibri';
 import { browser, passesRequirements } from './browserInfo';
 import plugin_data from 'plugin_data';
 
-const minimumBrowserRequirements = {
-  IE: {
-    major: 11,
-  },
-  Android: {
-    major: 4,
-    minor: 0,
-    patch: 2,
-  },
-};
+const minimumBrowserRequirements = {};
+
+const browserRegex = /^([a-zA-Z]+) ([><=]+) (\d+)(?:\.(\d+))?(?:\.(\d+))?$/;
+
+for (const browser of browsers) {
+  const [name, sign, major, minor, patch] = browserRegex.exec(browser).slice(1);
+  // This only supports > and >=, but that's all we need.
+  const addOne = sign === '>' ? 1 : 0;
+  const entry = {
+    major: isUndefined(minor) && isUndefined(patch) ? Number(major) + addOne : Number(major),
```

Comment:
My other comment as part of the review seems to have disappeared, so returning to this to add it in. 

The combination of the ternary operator, and the fact that `addOne` can be `1` or `0` (and isn't actually always 1, makes this hard for me to reason about. I think either adding a comment with a practical example, or adjusting the name of `addOne` (I have been trying to come up with a specific suggestion and have not so far, but will continue to ponder) might help make this clearer